tester_app/client/assets/stylesheets/_bootstrap-variables-customization.scss

```diff
@@ -0,0 +1,18 @@
+// This file is used as part of the bootstrap-sass-loader customization
+// Included from bootstrap-sass.config.js
```

Comment:
ah yes -- need some better explanation of this...Something's not right, as we should be including this in the file that rails uses as well! See other comment I just added.

We make this one called "react_on_rails_sass_helper", and have this loaded from the file with this name `_bootstrap-variables-customization.scss` and we include it in the bootstrap manifest for rails as well! perfecto!